packages/domain-picker/src/hooks/use-domain-suggestions.ts

```diff
@@ -32,7 +32,11 @@ export function useDomainSuggestions(
 		invalidateResolutionForStoreSelector: ( selectorName: string ) => void;
 	};
 
-	return useSelect(
+	const retryRequest = (): void => {
+		invalidateResolutionForStoreSelector( '__internalGetDomainSuggestions' );
+	};
```

Comment:
This function is basically being recalculated inside `useSelect`, but it doesn't make sense to be inside it, since it's just a simple wrapper around a `useDispatch` call. Moving it out.